Backend/handlers/handlers.go

```diff
@@ -123,3 +123,36 @@ func DeleteFriendshipHandler(c *gin.Context) {
 	deleter := c.Param("deleter")
 	c.JSON(http.StatusOK, friends.DeleteFriend(friendToDelete, deleter))
 }
+
+func EditNameHandler(c *gin.Context){
+
+	userID, _ := strconv.Atoi(c.Param("id"))
```

Comment:
I think it might be a good idea to change this to "username" instead, since the username is what will come from the frontend